packages/@n8n/config/src/decorators.ts

```diff
@@ -6,13 +6,25 @@ import { Container, Service } from 'typedi';
 type Class = Function;
 type Constructable<T = unknown> = new (rawValue: string) => T;
 type PropertyKey = string | symbol;
+type PropertyType = typeof Number | typeof Boolean | typeof String | Class;
```

Comment:
Is `typeof Number` somehow different from `number`? Or `Boolean` / `String`? Could we use the primitive ones instead?